Create ui-c8y-1024-7-0-device-text-based-configuration-tab-replace-textarea-with-c-8-y.md (#4953)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1024-7-0-device-text-based-configuration-tab-replace-textarea-with-c-8-y.md

---
date: ""

> title: Device text-based configuration tab now uses code editor

Reviewer: ```suggestion
title: Text-based configuration tab now uses code editor
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-6360
version: 1024.7.0
---

> The text-based configuration tab in device details now uses the code editor instead of a plain text area, adding syntax highlighting and code formatting.

Reviewer: ```suggestion
The **Text-based configuration** on the **Configuration** tab in the device details now uses the code editor instead of a plain text area, adding syntax highlighting and code formatting.
```

Reviewer 2: But in the configuration tab there are two sub tabs the legacy text based and the configuration and I am talking about the legacy text-based - aka the text-based configuration in device detail's configuration's tab. It is a bit too much was also not sure how to phrase it.

Reviewer: Got it :-). I will re-phrase it.

> When editing device text-based configurations, you previously used a basic text area that lacked syntax highlighting and code formatting features. The device text-based configuration tab now uses the Cumulocity code editor, which provides syntax highlighting, better code formatting, and improved editing capabilities. This change enhances your editing experience by making it easier to write and review configuration code with visual feedback and better readability.

Reviewer 2: ```suggestion
The text-based configuration tab in device details now uses the code editor instead of a plain text area, adding syntax highlighting and code formatting.

```
